static/app/routes.tsx

```diff
@@ -2103,11 +2103,18 @@ function buildRoutes() {
       <IndexRoute component={errorHandler(OverviewWrapper)} />
       <Route path="views/:viewId/" component={errorHandler(OverviewWrapper)} />
       <Route path="searches/:searchId/" component={errorHandler(OverviewWrapper)} />
+      <Redirect from=":groupId/tags/" to="/issues/:groupId/distributions/" />
+      <Redirect
+        from=":groupId/tags/:tagKey/"
+        to="/issues/:groupId/distributions/:tagKey/"
+      />
       <Route
         path=":groupId/"
         component={make(() => import('sentry/views/issueDetails/groupDetails'))}
         key="org-issues-group-id"
       >
+        {/* <Redirect from="tags/" to="/issues/:groupId/distributions/" />
+        <Redirect from="tags/:tagKey/" to="/issues/:groupId/distributions/:tagKey/" /> */}
```

Comment:
Putting the redirects here doesn't seem to work. They've gotta be outside of `<Route path=":groupId/" ...>`